src/App.js

```diff
@@ -41,6 +44,7 @@ import ProtectedRoute from './Components/ProtectedRoute';
 
 
 
+
```

Comment:
Remove Extra Spaces

Reply:
Spaces removed and code formatted according to the standards

Reply:
Noted..!!